collections/_release_4_4/entry-systems-animation-animation-markers.md

```diff
@@ -0,0 +1,23 @@
+---
+type: entry
+section: systems
+subsection: animation
+rank: 2
+importance: 3
+anchor: "animation-markers"
+title: "Animation markers"
+blockquote: "Put a pin in it"
+text: |
+  Markers allow you to create subregions of an animation that can be jumped to or looped without playing the entire animation.
+
+  These markers are also supported inside the Animation Tree, where you can easily select animation markers for looping or to begin playback.
```

Comment:
```suggestion
  These markers are also supported inside the Animation Tree, where you can easily create an AnimationNode’s custom timeline based on the markers.
```
As I pointed out previously in https://godotengine.org/article/dev-snapshot-godot-4-4-beta-1/, AnimationTree does not process playingback with markers directly.